tests/conftest.py

```diff
@@ -40,6 +41,13 @@ def fake_pdf():
     return FakePDF()
 
 
+@pytest.fixture
+def default_cards():
+    t = tc.generate(0, 1.0)
+    o = oc.generate([10.0])
```

Comment:
Use keyword arguments for all of them, it is not intuitive what is what.

Reply:
Even more: maybe we should make them only keyword, such that you can't use this way (prepending a `*` as first argument)